UDx-container/Dockerfile_RPM

```diff
@@ -11,119 +11,290 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-###########################################################################
+##############################################################################
 
 # Base operating system
 ARG os_version="7.9.2009"
 
-FROM centos:${os_version}
+FROM centos:${os_version} as builder
+
+ENV VERTICA_OPT_DIR="/opt/vertica"
+ENV VERTICA_VOLUME_DIR="/data"
+ENV VERTICA_DATA_DIR="${VERTICA_VOLUME_DIR}/vertica"
+ENV VERTICA_DB_NAME="vsdk"
+
+ARG vertica_db_user="dbadmin"
+ARG vertica_db_group="verticadba"
+ARG dbadmin_gid=1000
+ARG dbadmin_id=1000
 
+# name of .rpm file
 ARG RPM
 ARG vertica_version
 
-RUN bash -c "if [ \"$(rpm -E %{rhel})\" == '8' ]; then yum install -q -y dnf-plugins-core; yum -q config-manager --set-enabled powertools; yum -q -y install glibc-locale-source; fi" \
-    && yum -q -y update \
+COPY $RPM /tmp/$RPM
+
+# this is a script which removes unnecessary stuff from the
+# container image
+COPY ./tools/cleanup.sh /tmp/
```

Comment:
When I tried to create the container it failed here because `/tools/cleanup.sh` was missing.  Did we miss some files in this PR?

Reply:
Will add them